packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -383,7 +383,7 @@ class IOSDevice extends Device {
   @override
   DevicePortForwarder get portForwarder => _portForwarder ??= IOSDevicePortForwarder(
     processManager: globals.processManager,
-    logger: globals.logger,
+    logger: _logger,
```

Comment:
Might as well use the existing logger

Reply:
Good catch; this was an oversight on my part, but this is a REALLY long class...

Reply:
Yeah its too big!